core/setup.cfg

```diff
@@ -45,7 +45,9 @@ keywords =
     android
     windows
     winforms
+    unix-like
```

Comment:
In case it wasn't clear - these "unix-like" keywords aren't needed. It's not a search term anyone will be looking for.

Reply:
Sorry, I dropped the comments for 2 and 3, however the next commit will address them.